### ISSUE TITLE
RDP Wrap is not working on Windows 8.1 Pro - RDP session starts and then quits 5 seconds later

### ISSUE THREAD

**Issue by CharlieF2** [binarymaster/rdpwrap#32]
I've used RDP Wrap successfully on many different computers of all types.  

I've never had any issue before now.  

The PC is running Windows 8.1 Professional and the Terminal Services version is 6.3.9600.17415.  

I have removed the AVG anti-virus completely from the system and it is now only using the default Windows Defender - which I've used before without any issue.  

What is happening is that we are able to use a Remote Desktop Connection to connect to the "server" PC and we are presented with the normal User Log In screen and we are able to actually log into to the server as the user - briefly.  The desktop session is created and the desktop is displayed completely - but after about 5 seconds a message box opens and explains that there is a problem and that the sessions needs to close.  I don't have the exact message text but that is the gist of the message.  

I have tried changing the desktop display from a photo to a solid color - thinking that the image could be somehow causing the session to quit.  I've checked all of the normal permissions - remote desktop is allowed and we are not requiring NLA.  Each user is an Administrator and also a member of the Remote Desktop Users group.  

It is almost like there is something that is realizing that RDP is running and then it is being shutdown by something - without explanation.  

Has anyone seen this?  I would LOVE a prompt answer as I really need to get this working asap - thank you to everyone in advance for your assistance.  

Charlie

**Comment by CharlieF2** [comment]
And by the way I have added the entire RDP Wrapper folder to the list of files and folders that should be excluded from the Windows Defender and restarted the "server" PC and this has not resolved the issue either.  Can someone please explain where / how to implement the following suggestion that was made to another user in another entry?  

Try to use this command to isolate RDP Wrapper from other shared services:
sc config TermService type= own

Where would I use this command?  In the DOS window?  In the Run box?  Do I need to have elevated rights to the "Run as Administrator"

Charlie

**Comment by CharlieF2** [comment]
Hello everyone - so I didn't hear any responses from anyone here so I continued to research this issue on the web.  Happily I found others with the same general issue using RDP in general and not necessarily using RDP Wrap.  I found three possible solutions to this issue and the first one worked for me but I'm posting all of them here just in case someone else needs to know them.  Thank you for a great product and hopefully there won't be any more issues like this one to solve.  

Solution 1: To solve problem remove printers from local resources from you're remote desktop - the reason being either a corrupt or incompatible printer driver is being connected to over the RDP session and it is unable to do so and thus the connection crashes unexpectedly - and AFTER the initial RDP session started.  (This worked for me).  

Solution 2: "I had this problem on a Windows 7 machine and uninstalled update KB2830477 which is an update to the remote desktop client. "

Solution 3: What really fixed it for me was this:

1.Stop the "Remote Desktop Services"
3.Install update 2667402
4.Restart the computer or just "Remote Desktop Services"

Microsoft Reference for the fix:

http://www.microsoft.com/en-us/download/details.aspx?id=29145
